### PR TITLE
Project created

### PR DESCRIPTION
**Pull request by dharmeshec5**
Project Created

### CODE REVIEW

File: Product/ProductTests/ProductTests.swift [hunk 1]
```diff
@@ -0,0 +1,34 @@
+//
+//  ProductTests.swift
+//  ProductTests
+//
+//  Created by Dharmesh Patel on 05/04/20.
```
Comment (priyace5) [inline Product/ProductTests/ProductTests.swift]: Please change the name

**Review by priyace5 — CHANGES_REQUESTED** [review]

File: Product/ProductTests/ProductTests.swift (hunk 1 above)
Comment (dharmeshec5) [reply Product/ProductTests/ProductTests.swift]: This name is okay what is the issue in the name?

File: Product/ProductUITests/Info.plist [hunk 2]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
```
Comment (priyace5) [inline Product/ProductUITests/Info.plist]: Please remove UItest target

File: Product/ProductTests/ProductTests.swift [hunk 3]
```diff
@@ -0,0 +1,34 @@
+//
+//  ProductTests.swift
```
Comment (priyace5) [inline Product/ProductTests/ProductTests.swift]: Please remove this file

**Review by priyace5 — CHANGES_REQUESTED** [review]

File: Product/ProductTests/ProductTests.swift (hunk 3 above)
Comment (dharmeshec5) [reply Product/ProductTests/ProductTests.swift]: Done

File: Product/ProductUITests/Info.plist (hunk 2 above)
Comment (dharmeshec5) [reply Product/ProductUITests/Info.plist]: Done

**Review by priyace5 — APPROVED** [review]

**Pull request merged**